scripts/validate_ci_status_gates.py

```
@@ -0,0 +1,186 @@
+      - run: jq -e 'all(. as $result | ["success", "cancelled"] | any($result == .))'
+""",
+            [],
+        ),
+    ]
+
+    failures = []
+    for label, text, expected in cases:
+        errors = []
+        check_workflow("test.yaml", text, errors)
+        if len(errors) != len(expected) or not all(
+            fragment in error for fragment, error in zip(expected, errors)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)